nginx/conf.d/app.conf

```diff
@@ -0,0 +1,20 @@
+server {
+    listen 80;
```

Comment:
these are the configurations that `nginx` will run in the docker image, it will run the PHP and Javascript projects without having to run a server for them (nginx is the server)

What I mean, we don't need to do `php artisan serve` or `npm run dev` anymore